frontend/src/app/modules/bim/ifc_models/bcf/list-container/bcf-list-container.component.sass

```diff
@@ -0,0 +1,35 @@
+@import "helpers"
+
+.result-overlay
+  @include overlay-background
+  background-color: #FFFFFF
+  opacity: 0.5
+
+.work-packages--card-view-container
+  width: 100%
+  overflow: auto
+  padding-bottom: 5px
+  @include styled-scroll-bar
+
+  @media screen and (min-width: 679px)
```

Comment:
These min-width values looks to me as if they should be variables instead. What is this breakpoint? Can we give it a name?

Reply:
Previously this component shared template with the wp-list-view.component.ts, I just copied its template and styles and applied some modifications. These styles were already there so I'm not sure.

I see this same breakpoint used multiple times on the project so it seems a good idea to replace it with a variable, but maybe in its own pull request?

Reply:
OK with me.